lib/uwb/protocols/fira/StaticRangingInfo.cxx

```diff
@@ -0,0 +1,19 @@
+
+#include <iomanip>
+#include <sstream>
+#include <uwb/protocols/fira/StaticRangingInfo.hxx>
+
+std::string
+uwb::protocol::fira::StaticRangingInfo::ToString() const
+{
+    std::ostringstream ss;
+    ss << "{" << std::endl;
+    ss << "VendorId: " << VendorId << std::endl;
+    ss << "InitializationVector: " << std::hex << std::setw(2) << std::setfill('0') << std::showbase << std::internal;
+    for (const auto i : InitializationVector) {
+        ss << +i << " ";
+    }
+    ss << std::endl;
+    ss << "}" << std::endl;
```

Comment:
I'd prefer that these separators are only added for cases where there is structure nesting. This way when it's printed on its own, the caller can choose to insert their own separator chars between major structures if they so choose.

```suggestion
    ss << "VendorId: " << VendorId << std::endl;
    ss << "InitializationVector: " << std::hex << std::setw(2) << std::setfill('0') << std::showbase << std::internal;
    for (const auto i : InitializationVector) {
        ss << +i << " ";
    }
    ss << std::endl;
```